cglims/cli/commands.py

```diff
@@ -77,19 +77,22 @@ def config(context, gene_panel, family_id, samples, capture_kit, customer_or_cas
 @click.command()
 @click.option('-c', '--condense', is_flag=True, help='condense output')
 @click.option('-p', '--project', is_flag=True, help='identifier is a project')
+@click.option('-n', '--name', is_flag=True, help='identifier is the customer sample name')
```

Comment:
is this the most intuitive name for this option?
"getting" a sample is the default already so maybe "--external"? Just since it's an internal tool, the "name" of a sample should maybe refer to the LIMS id, or what do you think?